samples/net/zperf/prj_quark_se_c1000_devboard.conf

```diff
@@ -15,6 +15,10 @@ CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=2
 CONFIG_NET_MAX_CONTEXTS=10
 CONFIG_NET_CONTEXT_SYNC_RECV=y
 
+# Use all the RAM and ROM for x86 core
+CONFIG_RAM_SIZE=79
+CONFIG_ROM_SIZE=296
```

Comment:
How were these numbers arrived at?
AFAIK the board has 80K of RAM and 384K of flash (although a chunk of flash is used for 2nd stage DFU loader)

Reply:
From boards/arc/quark_se_c1000_ss_devboard/quark_se_c1000_ss_devboard.dts
#define DT_FLASH_SIZE		__SIZE_K(152)
#define DT_SRAM_SIZE		__SIZE_K(24)

And from boards/x86/quark_se_c1000_devboard/quark_se_c1000_devboard.dts
#define DT_FLASH_SIZE		__SIZE_K(144)
#define DT_SRAM_SIZE		__SIZE_K(55)

Adding these two I got 152 + 144 = 296 and 24 + 55 = 79. Perhaps the number are originally wrong in the dts file or I am doing something totally wrong here. I sent this patch as sanity complained that the traffic class PR #6076 ran out of mem for this board.